polars/polars-core/src/chunked_array/ops/aggregate.rs

```diff
@@ -288,7 +288,7 @@ where
     }
     fn mean_as_series(&self) -> Series {
         let s = self.sum_as_series();
-        let mut out = s.cast::<Float64Type>().unwrap() / self.len() as f64;
+        let mut out = s.cast::<Float64Type>().unwrap() / (self.len() - self.null_count()) as f64;
```

Comment:
This is obviously an edge case, but do we need to worry about a `Series` that contains _all_ null values (I would think this should return `NaN`, and I'm not sure if this will raise an error instead)?

Reply:
You are right! I just didn't think of that.